modules/nsgaii.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, gen: int, element_simulation: ElementSimulation = None,
                  recoil_type="box", number_of_processes=1, cross_p=0.9, mut_p=1,
                  stop_percent=0.3, check_time=20, ch=0.025,
                  measurement=None, cut_file=None, dis_c=20,
-                 dis_m=20, check_max=900, check_min=0, skip_simulation=False,
+                 dis_m=20, check_max=900, check_min=0, skip_simulation=True,
```

Comment:
A unit test fix shouldn't change the default parameters for actual code.

Pass `skip_simulation=True` in the test itself, or change the expected value for the test.

Reply:
Done